src/messages/serenity.proto

```diff
@@ -7,10 +7,25 @@ option java_outer_classname = "Protos";
 
 
 /**
-* Necessary union to define job.
-* Currently we cannot specify individual task.
-* NOTE: Framework id must be set for the specified executor.
-*/
+ * Events Envelopes for EventBus.
+ *
+ * TODO(bplotka): Add topic for EventBus and do not specify aim of the
+ * event by message Type, but by the topic.
+ */
+message OversubscriptionReCtrlEventEnvelope {
```

Comment:
I think we don't need both RE and QoS messages.
We should stick with OversubscriptionCtrlEventEnvelope that will contain OversubscriptionCtrlEvent{bool msg}, and state that bus should be only used for inter-pipeline communication.